mozetl/taar/taar_amowhitelist.py

```diff
@@ -64,12 +64,24 @@ def transform(self, json_data):
 
         new_data = {}
         for guid in json_data.keys():
-            rating = json_data[guid]['ratings']['average']
-            create_date = parse(json_data[guid]['first_create_date']).replace(tzinfo=None)
+            addon_data = json_data[guid]
+
             if guid == 'pioneer-opt-in@mozilla.org':
                 # Firefox Pioneer is explicitly excluded
                 continue
 
+            current_version_files = addon_data.get('current_version', {}).get('files', [])
+            if len(current_version_files) == 0:
+                # Only allow webextensions
+                continue
+            else:
+                if current_version_files[0].get('is_webextension', False) is False:
+                    # Only allow webextensions
+                    continue
+
+            rating = addon_data['ratings']['average']
```

Comment:
Let's be consistent with the code above and use `.get()` whenever possible, eventually catching errors and failing gracefully.